apps/communications/contacts/js/match_service.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+/* global utils */
+
+(function(exports) {
+  var matchService = {};
+
+  function setupFragment() {
+    var extensions = document.createElement('iframe');
+    extensions.id = 'match-extension';
+    document.body.appendChild(extensions);
+  }
+
+  setupFragment();
+
+  var CONTACTS_APP_ORIGIN = location.origin;
+  var extensionFrame = document.querySelector('#match-extension');
```

Comment:
If you dont wrap the creation of the iframe as a function, you can use directly the `<iframe>` created previously.